packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -132,6 +132,109 @@ void main() {
     expect(material.elevation, elevation);
     expect(material.shape, shape);
   });
+
+  testWidgets('BottomSheet modalElevation takes priority over elevation - Modal', (WidgetTester tester) async {
+    const double modalElevation = 5.0;
+    const double persistentElevation = 7.0;
+    const BottomSheetThemeData bottomSheetTheme = BottomSheetThemeData(
+      elevation: persistentElevation,
+      modalElevation: modalElevation,
+    );
+
+    await tester.pumpWidget(bottomSheetWithElevations(bottomSheetTheme));
+    await tester.tap(find.text('Show Modal'));
+    await tester.pumpAndSettle();
+
+    final Material material = tester.widget<Material>(
+      find.descendant(
+        of: find.byType(BottomSheet),
+        matching: find.byType(Material),
+      ).first,
+    );
+    expect(material.elevation, modalElevation);
+  });
+
+  testWidgets('BottomSheet elevation takes priority over modalElevation - Persistent', (WidgetTester tester) async {
+    const double modalElevation = 5.0;
+    const double persistentElevation = 7.0;
+    const BottomSheetThemeData bottomSheetTheme = BottomSheetThemeData(
+      elevation: persistentElevation,
+      modalElevation: modalElevation,
+    );
+
+    await tester.pumpWidget(bottomSheetWithElevations(bottomSheetTheme));
+    await tester.tap(find.text('Show Persistent'));
+    await tester.pumpAndSettle();
+
+    final Material material = tester.widget<Material>(
+      find.descendant(
+        of: find.byType(BottomSheet),
+        matching: find.byType(Material),
+      ).first,
+    );
+    expect(material.elevation, persistentElevation);
+  });
+
+  testWidgets('BottomSheet modalElevation only sets elevation for modal bottom sheet - Persistent', (WidgetTester tester) async {
+    const double modalElevation = 5.0;
+    const BottomSheetThemeData bottomSheetTheme = BottomSheetThemeData(
+      modalElevation: modalElevation,
+    );
+
+    await tester.pumpWidget(bottomSheetWithElevations(bottomSheetTheme));
+    await tester.tap(find.text('Show Persistent'));
+    await tester.pumpAndSettle();
+
+    final Material material = tester.widget<Material>(
+      find.descendant(
+        of: find.byType(BottomSheet),
+        matching: find.byType(Material),
+      ).first,
+    );
+    expect(material.elevation, 0);
+  });
+}
+
+Widget bottomSheetWithElevations(BottomSheetThemeData bottomSheetTheme) {
+  return MaterialApp(
+    theme: ThemeData(bottomSheetTheme: bottomSheetTheme),
+    home: Scaffold(
+      body: Builder(
+        builder: (BuildContext context) {
+          return Column(
+              children: <Widget>[
+                RawMaterialButton(
+                  child: const Text('Show Modal'),
+                  onPressed: () {
+                    showModalBottomSheet<void>(
+                      context: context,
+                      builder: (_) => Container(
```

Comment:
Please reserve the `=>` notation for one-liners that return a value (here and elsewhere).

In cases like this, where you'd like to indicate that you're ignoring a parameter, please include the parameter type:
`builder: (BuildContext _)`

Reply:
Done